solutions/cses-1661.mdx

```diff
@@ -2,24 +2,29 @@
 id: cses-1661
 source: CSES
 title: Subarray Sums II
-author: Qi Wang
+author: Qi Wang, Brad Ma
 ---
 
-# Problem
+## Problem
 
 We are asked to find the number of subarrays that sum up to $x$ given the size
 of the array and its elements.
 
-<Spoiler title="Solution">
+## Explanation
 
 We can have a map that keeps track of the prefix sums. At each index $i$, we can
 count the number of prefixes with sum equal to $\texttt{prefixSum}[i] - x$. This
```

Comment:
I feel like using the word prefix isn't specific enough. Maybe do it in terms of \sum{0...i} or something

Reply:
lol ask harry for that i just wrote the java solution